### PR TITLE
use overwriting config when running included apps

### PR DESCRIPTION
**Pull request by calvinalkan**
See: #5978

### CODE REVIEW

File: tests/data/included/jazz/tests/_helpers/CustomReporter1.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Jazz;
+
+use Codeception\PHPUnit\ResultPrinter\Report;
+
+final class CustomReporter1 extends Report
+{
+    public function endTest(\PHPUnit\Framework\Test $test, float $time) :void
```
Comment (Naktibalda) [inline tests/data/included/jazz/tests/_helpers/CustomReporter1.php:9]: Please come up with the test which doesn't use custom reporters, this test won't work on Codeception 5.

**Review by Naktibalda — CHANGES_REQUESTED** [review]

File: tests/data/included/jazz/tests/_helpers/CustomReporter1.php (hunk 1 above)
Comment (calvinalkan) [reply tests/data/included/jazz/tests/_helpers/CustomReporter1.php:9]: Any suggestion? Colors?

Comment (calvinalkan) [reply tests/data/included/jazz/tests/_helpers/CustomReporter1.php:9]: What else could be asserted in output?

Comment (Naktibalda) [reply tests/data/included/jazz/tests/_helpers/CustomReporter1.php:9]: Fine, you will need it for Codeception 4 only anyway :)

File: tests/cli/IncludedCest.php [hunk 2]
```diff
@@ -205,4 +205,19 @@ public function includedSuitesAreNotRunTwice (CliGuy $I) {
...
         $I->dontSeeInShellOutput('4 tests');
     }
+    
+    /**
+     * @param CliGuy $I
+     * @before moveToIncluded
+     */
+    public function overwrittenConfigurationIsAlsoAppliedWhenRunningAnIncludedAppFromTheRootApp(CliGuy $I)
+    {
+        if(PHP_VERSION_ID < 700100) {
```
Comment (Naktibalda) [inline tests/cli/IncludedCest.php:215]: Correct version id is 70100

**Comment by Naktibalda** [comment]
Merged to 4.1 https://github.com/Codeception/Codeception/commit/b815ea3fa7c2cf54b96cb68bf93b528e67ffb63f

**Comment by Naktibalda** [comment]
This change will be released as 4.2.0 when #6435 is merged.

**Pull request closed without merging**